package.json

```diff
@@ -57,7 +57,7 @@
     "passport-twitter": "^1.0.2",
     "phantomjs": ">=1.9.0",
     "serve-favicon": "^2.3.0",
-    "socket.io": "^1.3.5",
+    "socket.io": "Automattic/socket.io",
```

Comment:
@bastianwegge Yes. I saw that. I was hoping that we could install the stand-alone *HEAD* version of socket.io-client, and still use the version installed with socket.io. Guess that's not the case, as Cody pointed out. I say we sit tight and work on these tests in the meantime.